pkgs/applications/version-management/sourcehut/fix-gqlgen-trimpath.nix

```diff
@@ -0,0 +1,32 @@
+{ unzip }:
+{
+  overrideModAttrs = (_: {
+    # No need to workaround -trimpath: it's not used in go-modules,
+    # but do download `go generate`'s dependencies nonetheless.
+    preBuild = ''
+      set -x
```

Comment:
```suggestion
```
We shouldn't leave debugging code in.

Reply:
I would have preferred to have this tracing on because it was not obvious where Go was failing, but I don't feel strongly about it so let's run without it until the next breakage ¯\_(ツ)_/¯